waves-ext/src/main/scala/com/wavesplatform/dex/grpc/integration/services/WavesBlockchainApiGrpcService.scala

```diff
@@ -180,6 +189,14 @@ class WavesBlockchainApiGrpcService(context: ExtensionContext)(implicit sc: Sche
           CheckedBroadcastResponse(CheckedBroadcastResponse.Result.Failed(CheckedBroadcastResponse.Failure(message, canRetry = false)))
       }
 
+  private def handleTxInUtx(tx: exchange.ExchangeTransaction): Future[CheckedBroadcastResponse.Result] =
+    context.transactionsApi.broadcastTransaction(tx).map { //todo: check with tests
+      _.resultE match {
+        case Right(isNew) => CheckedBroadcastResponse.Result.Unconfirmed(isNew)
+        case Left(e) => CheckedBroadcastResponse.Result.Failed(CheckedBroadcastResponse.Failure(e.toString, canRetry(e)))
```

Comment:
1. We need to return `Unconfirmed(false)`
2. If we have an error we just log it

Reply:
Fixed